gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -252,6 +252,7 @@ def volume_service(tenant_name = nil)
       connect(:service => "Volume", :tenant_name => tenant_name)
     end
     alias_method :connect_volume, :volume_service
+    alias_method :cinder_service, :volume_service
```

Comment:
Here again, these are not the same. The cinder_service method needs to ensure the volume service is indeed Cinder.